frontend/src/renderer/components/About.svelte

```diff
@@ -1,101 +1,283 @@
-        are also written cues describing the action to take on each screen
-        within the application as it is navigated
-      </li>
-    </ol>
-    <br />
-    <p>
-      Beyond RGB was developed by Patrick Dalesio, Andrew Hammerstone, Tristan
-      Knox, Jake O’Neil, and Jared Ponzetti. Thank you for the time and talent
-      you devoted to creating this tool. It really is better than Really Good
-      Bread.
-    </p>
+  <div class="modal-header" in:fly={{ y: -20, duration: 400, delay: 100 }}>
+    <h1 class="title">About Beyond RGB</h1>
+    <p class="subtitle">Imaging Art beyond RGB</p>
   </div>
+  
+  <ScrollContainer maxHeight="65vh" className="body">
+    <div class="content-wrapper" in:fade={{ duration: 400, delay: 200 }}>
+      <!-- Overview Section -->
+      <section class="info-section">
+        <div class="section-header">
+          <h2>Overview</h2>
+        </div>
+        <p class="section-text">
+          Beyond RGB is an open-source software application for processing spectral
+          image sets. It takes as input two RAW (or unprocessed linear TIFF) RGB
+          image sets (object/flatfield/darkfield) captured under two different
+          lighting or filtering conditions, and outputs:
+        </p>
+        <div class="feature-list">
+          <div class="feature-item">
+            <CheckCircleIcon size="1x" class="feature-icon" />
+            <span>A color managed RGB image</span>
+          </div>
+          <div class="feature-item">
+            <CheckCircleIcon size="1x" class="feature-icon" />
+            <span>A spectral reflectance transform that enables the interactive estimation
+              of reflectance curves from user-selected regions of interest in the image</span>
+          </div>
+          <div class="feature-item">
+            <CheckCircleIcon size="1x" class="feature-icon" />
+            <span>Two sets of ancillary data and plots detailing the accuracy of the color
+              and spectral calibrations, respectively</span>
+          </div>
+        </div>
+      </section>
+
+      <!-- Design Philosophy Section -->
+      <section class="info-section">
+        <div class="section-header">
+          <h2>Design Philosophy</h2>
+        </div>
+        <p class="section-text">
+          Beyond RGB was designed with considerations for form, function, and user
+          friendliness, and is intended for use by cultural heritage imaging
+          professionals. It is cross-platform compatible and operable via an
+          interactive graphical user interface, and will continue to be a living,
+          updatable open-source project.
+        </p>
+        <div class="contact-box">
+          <p class="contact-text">
+            It is freely available for download from the RIT Munsell Color Science Laboratory website.
+          </p>
+          <p class="contact-email">
+            Please direct comments and questions to <strong>susan.farnand@rit.edu</strong>
```

Comment:
this would open the user's set email application, and most places that have emails usually open as a `mailto` link
```suggestion
            Please direct comments and questions to <a href="mailto:susan.farnand@rit.edu">susan.farnand@rit.edu</a>
```